codespell_lib/data/dictionary.txt

```diff
@@ -16907,6 +17022,8 @@ expcetation->expectation
 expcetations->expectations
 expceted->expected
 expceting->expecting
+expcetion->exception
```

Comment:
Mmmh... Shouldn't this be:
```
expcetation->expectation
```
in addition to:
```
expcetion->exception
```

Reply:
`expcetation->expectation` exists a few lines earlier.

As for this entry, my script grabbed `expcet->except` from `expcet->expect, except,` and then added the suffix `-ion`.